scripts/lib/CIME/utils.py

```diff
@@ -254,6 +254,27 @@ def get_cime_root(case=None):
     logger.debug( "CIMEROOT is " + cimeroot)
     return cimeroot
 
+def get_src_root(case=None):
+    """
+    Return the absolute path to the root of SRCROOT.
+
+    """
+    script_absdir = os.path.abspath(os.path.join(os.path.dirname(__file__),".."))
+    assert script_absdir.endswith(get_python_libs_location_within_cime()), script_absdir
```

Comment:
What's the point of these two lines?

Reply:
No point, I just copied get_cime_root.  I'll remove it.